global_install_scripts/talisman_hook_script.bash

```diff
@@ -78,8 +77,12 @@ fi
 
 DEBUG_OPTS=""
 [[ $(toLower "${TALISMAN_DEBUG}") == "true" ]] && DEBUG_OPTS="-d"
+
 INTERACTIVE=""
-[[ $(toLower "${TALISMAN_INTERACTIVE}") == "true" ]] && INTERACTIVE="-i"
+if [ $(toLower "${TALISMAN_INTERACTIVE}") == "true" ]; then
+    INTERACTIVE="-i"
+	  [[ "${HOOKNAME}" == "pre-commit" ]] && exec < /dev/tty || echo_warning "talisman pre-push hook cannot be invoked in interactive mode currently"
+fi
```

Comment:
Hey there, just noticed that since you've change the `[[ ... ]]` to `if [ ... ]`, `$(toLower "${TALISMAN_INTERACTIVE}")` now needs to be quoted, otherwise you will get 

```txt
.git/hooks/bin/pre-commit: line 82: [: ==: unary operator expected
```

if `$TALISMAN_INTERACTIVE` is not set. 

Easy mistake to make :P
```bash
if [ "$(toLower "$TALISMAN_INTERACTIVE")" == "true" ]; then
```

does the trick!